Code/GraphMol/Wrap/Atom.cpp

```diff
@@ -141,6 +141,9 @@ AtomPDBResidueInfo *AtomGetPDBResidueInfo(Atom *atom) {
   return (AtomPDBResidueInfo *)res;
 }
 
+struct MDLDummy {};
```

Comment:
Additionally, isn't the MolFileValue where the recursive smarts is stored in a mol block?  I think this is probably good to document perhaps.